examples/UDPServer.py

```diff
@@ -0,0 +1,11 @@
+import socket
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+sock.bind(("127.0.0.1", 12345))
+
+while True:
+    data, addr = sock.recvfrom(4096) #For sending data, use recv_from
+    print(str(data))
+    print(str(addr))
+    msg = b"Hello, I am a UDP Server!"
+    sock.sendto(msg, addr)

```

Comment:
This never returns, and thus you have to kill your terminal to get it to exit. Maybe look at this for a solution https://www.kite.com/python/answers/how-to-handle-a-sigterm-interrupt-in-python